tests/components/teslemetry/test_sensor.py

```diff
@@ -26,7 +26,8 @@ async def test_sensors(
 ) -> None:
     """Tests that the sensor entities are correct."""
 
-    freezer.move_to("2024-01-01 00:00:00+00:00")
+    # Use local timezone
+    freezer.move_to(datetime(2024, 1, 1, 0, 0, 0))
```

Comment:
I think freezer had a tz parameter?

Reply:
I am trying to avoid setting a timezone though, because @emontnemery was having failures on his machine because the sensor datetime didnt match the snapshot.